srcpkgs/discord-ptb/template

```diff
@@ -1,19 +1,31 @@
 # Template file for 'discord-ptb'
-pkgname="discord-ptb"
-version=0.0.9
+pkgname=discord-ptb
+version=0.0.21
 revision=1
+archs="x86_64"
 wrksrc="DiscordPTB"
 depends="alsa-lib dbus-glib gtk+3 GConf libnotify nss libXtst libcxx libatomic"
-archs="x86_64"
-nopie=yes
-repository=nonfree
-restricted=yes
 short_desc="Proprietary freeware VoIP application"
 maintainer="Abel Graham <abel@abelgraham.com>"
-license="proprietary"
+license="custom:Proprietary"
 homepage="https://discordapp.com/"
 distfiles="https://dl-ptb.discordapp.net/apps/linux/${version}/discord-ptb-${version}.tar.gz"
-checksum=372d6f75203626370218c97c21519b779e5af334177cafa65a5655e357c77520
+checksum=a1fa4e20cce5f493d84c9e3c7ac74c8265768886859e98022fa41d102f2726c4
+_license_checksum=e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+nopie=yes
+restricted=yes
+repository=nonfree
+
+post_extract() {
+	curl -L https://discord.com/terms |
+		w3m -dump -I utf-8 -T text/html |
+		sed -n '/Discord Terms of Service/,/^USA$/p' > EULA
```

Comment:
There is no word "USA" in this license.

Reply:
You're right. I have no idea why I didn't replace that with something that's inside of the EULA.
Thanks for pointing it out!